CONTRIBUTING.md

```diff
@@ -14,6 +14,7 @@ Here's a few tips as you go along:
 # Translations
 
 * All articles may be translated into any language of interest.
+* Instead of translating from scratch, consider starting with [machine-translated text](https://www.deepl.com/translator) and revising from there.
```

Comment:
Added a  sentence stating what we heard from DeepL by mail.
```suggestion
* Instead of translating from scratch, consider starting with [machine-translated text](https://www.deepl.com/translator) and revising from there. You may remove the "translated by DeepL" line added in your pull request.
```